resources/js/components/OptOut/OptOut.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+<div class="main-content opt-out">
+
+        <div class="player-name">
+        <h2>Opt Out</h2>
+        </div>
+        <div class="standard-page-content">
+<section class="__reader_view_article_wrap_8379893062043087__">
+
+<p>All of the data on this site is dynamically generated by Blizzard data obtained by replays uploaded to HotsAPI. Please refer to Blizzard's licensing agreement regarding their ownership of your battletag/blizz_id.  Any players' replay they upload may have information on your assigned battletag and we do not control which data is uploaded to HotsAPI.</p>
+<p>If you do not wish for your battletag to appear on this site on Profile or Match pages, we provide the option for you to opt-out. In order to do this, we require that you authenticate with your Blizzard account. This ensures that you are the player with that blizz_id and ensures that the system is not abused by those wishing to remove other players' data.</p>
+<p>Please choose a region below and click "opt-out". You will be redirected to Blizzard's secure authentication and asked to allow access for your Blizz ID and Battletag. This is the only information we use, and we do not store this information anywhere. We do not collect or store any personal information. After you have opted out, you may remove your authorization for this website by going to your Battle.net account under "Security &amp; Privacy" and managing your authorized applications.</p>
+<p>Once you have opted-out, data uploaded from replays in which you played a game will still be used to calculate data across the site, but your Battletag and Profile page will be removed.</p>
+
+<!--Need to add a confirm check here before allowing them to opt out-->
+<p>Please be aware that opting out also removes your data for use by others using our API tool.  So if your MMR is required for Amateur series, this will cause your data not to show up for them, and may make you ineligable for league play.<p>
```

Comment:
This is the only thing I saw missing from the opt out page. 

The user needs to accept the above statements before they can click the opt out button